src/features/ProtectTransaction/utils.ts

```diff
@@ -81,4 +82,13 @@ export abstract class ProtectTxUtils {
       return false;
     };
   }
+
+  public static getTypeFromNansenLabels(labels: string[]): NansenReportType {
+    if (MALICIOUS_LABELS.some((l) => labels.includes(l))) {
+      return NansenReportType.MALICIOUS;
+    } else if (WHITELISTED_LABELS.some((l) => labels.includes(l))) {
+      return NansenReportType.WHITELISTED;
+    }
+    return NansenReportType.UNKNOWN;
```

Comment:
Do we really want to quit on the first malicious label vs displaying each one?

Reply:
This isn't for displaying it, but simply to detect if we should show the account as malicious or not. Every label should be shown either way 😄